core/codenvy-hosted-sso-server-organization/src/main/java/com/codenvy/auth/sso/server/OrgServiceUserValidator.java

```diff
@@ -64,6 +71,10 @@ public void ensureUserCreationAllowed(String email, String userName) throws Conf
             throw new BadRequestException("User name must contain letters and digits only");
         }
 
+        if (reservedNames.contains(userName)) {
+            throw new ConflictException("User name is reserved. Please, choose another one");
```

Comment:
include username -->
throw new ConflictException(String.format("User %s is reserved, Please, choose another one", userName))